<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginDialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="yspcclient.qrc">
    <normaloff>:/res/login/logo.png</normaloff>:/res/login/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#LoginDialog
{
border-image: url(:/res/login/login_background.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>199</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>310</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>35</number>
     </property>
     <item>
      <widget class="QWidget" name="WidgetLogin" native="true">
       <property name="minimumSize">
        <size>
         <width>771</width>
         <height>409</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>771</width>
         <height>409</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#WidgetLogin
{
border-image: url(:/res/login/login_login.png);
}</string>
       </property>
       <widget class="QPushButton" name="PButtonLogin">
        <property name="geometry">
         <rect>
          <x>395</x>
          <y>275</y>
          <width>280</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
color: rgb(255, 255, 255);
background-color: rgb(255, 85, 0);
border-radius:25px;
font-size:18px;
}</string>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="EditUser">
        <property name="geometry">
         <rect>
          <x>415</x>
          <y>143</y>
          <width>280</width>
          <height>34</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px;
border-bottom:1px solid rgb(222,222,222);
background:transparent;
</string>
        </property>
        <property name="text">
         <string>15071620224</string>
        </property>
        <property name="maxLength">
         <number>50</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="EditPassword">
        <property name="geometry">
         <rect>
          <x>415</x>
          <y>207</y>
          <width>280</width>
          <height>34</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0px;
border-bottom:1px solid rgb(222,222,222);
background:transparent;
</string>
        </property>
        <property name="text">
         <string>123456</string>
        </property>
        <property name="maxLength">
         <number>50</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="LabelLoginTitle">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>60</y>
          <width>310</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:30px;
color: rgb(55, 55, 55);</string>
        </property>
        <property name="text">
         <string>登录有师</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="LabelUser">
        <property name="geometry">
         <rect>
          <x>334</x>
          <y>143</y>
          <width>80</width>
          <height>34</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:15px;
color: rgb(111, 111, 111);</string>
        </property>
        <property name="text">
         <string>账号：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="LabelPassword">
        <property name="geometry">
         <rect>
          <x>334</x>
          <y>207</y>
          <width>80</width>
          <height>34</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:15px;
color: rgb(111, 111, 111);</string>
        </property>
        <property name="text">
         <string>密码：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PButtonLoadapp">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	border:2px solid rgb(255,80,0);
	background:none;
	color: rgb(255,80,0);
	font-size:20px;
	border-radius:25px;
}
</string>
       </property>
       <property name="text">
        <string>下载APP</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>309</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>199</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>EditUser</tabstop>
  <tabstop>EditPassword</tabstop>
  <tabstop>PButtonLogin</tabstop>
 </tabstops>
 <resources>
  <include location="yspcclient.qrc"/>
 </resources>
 <connections/>
</ui>
